src/main/java/com/mabl/integration/jenkins/MablRestApiClientImpl.java

```diff
@@ -153,6 +164,15 @@ private CredentialsProvider getApiCredentialsProvider(
 
         provider.setCredentials(AuthScope.ANY, creds);
 
+        // Set proxy credentials if provided
+        if (proxy != null && (!StringUtils.isBlank(proxy.getUserName()) || !StringUtils.isBlank(proxy.getPassword()))) {
+            final Credentials credentials = new UsernamePasswordCredentials(
+                    proxy.getUserName() == null ? "" : proxy.getUserName(),
```

Comment:
Is this null handler needed for passwords too?

Reply:
I looked at the implementation and it was not necessary.